milestone-3/frontend/src/App.js

```diff
@@ -0,0 +1,45 @@
+import Navbar from "./navbar";
+import recipes from "./recipeData";
+import RecipeHome from "./components/recipeHome";
+import RecipePage from "./components/recipePage";
+import About from "./components/about";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+
+function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Switch>
+        <Route exact path="/">
+          <h1 style={{ margin: "10px" }}>Welcome to Noah's Nibbles</h1>
+          <p style={{ margin: "40px" }}>
+            Are you ready to cook some tasty meals, look no further!
+          </p>
+          {recipes.map((recipe) => (
+            <RecipeHome
```

Comment:
by this logic, you would have 3 home pages, I think this Component could be better named like preview or something similar.